docs/guides/genomics-platform/requesting-data/about-our-data.md

```diff
@@ -50,9 +50,9 @@ Somatic VCF files available on St. Jude Cloud have been generated when possible
 
 The validation_status field indicates the status of validating the presence and origin (somatic/germline) of the variant using an orthogonal methodology. Note that this doesn't imply anything regarding variant significance, pathogenicity, or relation to disease. Typically, validation is performed either by secondary sequencing using a custom capture panel designed around the called variants (common in research studies) or by cross-validation using simultaneous sequencing by WGS, WES, and/or RNA-Seq (used in clinical genomics).
  
-For validation by a secondary method, "valid" indicates the variant was validated by the secondary method, "putative" indicates that the secondary method could neither confirm nor invalidate the variant (e.g. due to lack of read coverage by the secondary method), and "invalid" indicates a negative result from the secondary method.
+Validation by a secondary sequencing method has commonly been used in retrospective research studies such as PCGP. Here, "valid" indicates the variant was validated by the secondary method, "putative" indicates that the secondary method could neither confirm nor invalidate the variant (e.g. due to lack of read coverage by the secondary method), and "invalid" indicates a negative result from the secondary method.
```

Comment:
I think the generic "method" is more applicable here. You can validate by a supplemental sequencing method, but you could also validate by a laboratory test.  Further, I think we want to avoid being detailed in this paragraph as it describes the general concept of validating a variant. In the following two paragraphs, we discuss specific validation methods such as cross-validation for clinical sequencing and custom capture for research projects such as PCGP. I think this is why Michael R. structured the paragraphs this way.

Reply:
Yes, I was under the impression that all such PCGP validation methods only comprised further sequencing at each variant site (hence my edit). Though I'm happy for it to be changed back.